atd-vze/src/routes.js

```diff
@@ -63,6 +63,7 @@ const ToolsUploadNonCR3 = React.lazy(() =>
 const CreateCrashRecord = React.lazy(() =>
   import("./views/Tools/CreateCrashRecord")
 );
+const Page404 = React.lazy(() => import("./views/Pages/Page404/Page404"));
```

Comment:
I'm getting this warning
`  Line 66:7:  'Page404' is assigned a value but never used`

Reply:
Thanks yeah i dont need that line anymore and forgot to take it out!